snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/SnomedRf2ImportRequest.java

```diff
@@ -100,6 +101,9 @@
 	@JsonProperty
 	private boolean createVersions = true;
 	
+	@JsonProperty
+	private List<String> ignoreMissingReferencesIn = Lists.newArrayList();
```

Comment:
Use `Set` instead of `List` for this feature. Use `@NotNull` annotation instead of initializing it here. RequestBuilders should initialize their Request instances during `build()`.